noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1056,30 +1509,14 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Registry entries without a main document</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, r._dtype registry_entry_type, COUNT(*) registry_entries_without_a_main_document
-                    FROM arkivstruktur.registrering r
-                    WHERE r._dtype = 'journalpost' AND r.systemid NOT IN (
-                        SELECT _ref_registrering
-                        FROM arkivstruktur.dokumentbeskrivelse
-                        WHERE tilknyttetregistreringsom = 'Hoveddokument' AND _ref_registrering IS NOT NULL
-                        GROUP BY _ref_registrering
-                    )
-                    GROUP BY r._ref_arkivdel, r._dtype;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST11">
         <title>Record period containment</title>
+        <description>
+            Tests whether the Records' created and finalized dates are
```

Comment:
`Tests whether the created and finalized dates of all records are within the archival period specified in arkivuttrekk.xml`